### PR TITLE
Fixed an issue where tableHeaderView height settings were repeated.

### PR DESCRIPTION
**Pull request by daisuke-t-jp**
Task/Issue URL: https://app.asana.com/0/414235014887631/1201091852879772/f
Tech Design URL:
CC:

**Description**:
Hi,

The height setting of `tableHeaderView` is executed repeatedly.
This makes scrolling unnatural in some situations. Also, the toolbar may disappear.
I attach a video of the problem.

I fixed `tableHeaderView` height setting so that it doesn't repeat.

Thanks.

https://user-images.githubusercontent.com/45647069/133078551-43aaa983-fd3e-448d-a773-5ca8543553d5.mov

**Steps to test this PR**:
1. Open the following view controller screen.
    - DoNotSellSettingsViewController
    - UnprotectedSitesViewController
    - ReportBrokenSiteViewController
    - PrivacyProtectionEncryptionDetailController
    - PrivacyProtectionNetworkLeaderboardController
    - PrivacyProtectionPracticesController
    - PrivacyProtectionTrackerNetworksController
1. Scrolling
    - [x] Does not repeatedly set the height of tableHeaderView.
    - [x] Scrolling is natural.
    - [x] No layout issues.

**Copy Testing**:

* [x] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Device Testing**:

* [x] iPhone 8

**OS Testing**:

* [x] iOS 14

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**

### CODE REVIEW

**Comment by samsymons** [comment]
Thanks for your patience waiting for a review @daisuke-t-jp, we've been super busy. I'll try to get to this today. 🙏

**Review by samsymons — CHANGES_REQUESTED** [review]
This is such a nice change, thanks @daisuke-t-jp! 🙏 The issue with scrolling getting stuck is great to have fixed.

I've only noticed one issue, where the toolbar at the bottom of the Unprotected Sites screen no longer extends to the bottom of the screen and instead only goes to the safe area. Once that's sorted, this PR is good to merge!

Unprotected Sites

**Comment by daisuke-t-jp** [comment]
@samsymons 
Thanks your review.

This issue is occurring not only in `fix-table-header-view-resize` branch, but also in the `develop` branch.

I investigated this problem.
It occurred in the following ☑️ mark pattern.
In other words, it happens when build with Xcode13 and run it on iOS 15.0.

- [ ] Xcode 12 / iPhone 12 - iOS 14.5 Simulator
- [ ] Xcode 12 / iPhone 12 - iOS 15.0 Simulator
- [ ] Xcode 13 / iPhone 12 - iOS 14.5 Simulator
- [x] Xcode 13 / iPhone 12 - iOS 15.0 Simulator


I looked it up, but unfortunately I have no idea how to fix this problem.


Does this issue also occur on real iOS 15.0 devices?
I'm sorry, I don't have an iPhone device with a safeArea at the bottom of the screen. (iPhone X, 11, 12, 13)

Regards.

**Comment by samsymons** [comment]
@daisuke-t-jp You're absolutely right, so it does. This does occur on a real device, but given that this isn't caused by your changes I don't consider the issue a blocker - we'll track it separately. :+1:

**Review by samsymons — APPROVED** [review]
LGTM :+1:

**Pull request merged**
